app/templates/courses/teacher-class-view.jade

```diff
@@ -511,7 +511,7 @@ if classroom.loaded
     if !state.get('readOnly')
       a.label.edit-classroom(data-classroom-id=classroom.id)
         span(data-i18n='teacher.edit_class_settings')
-    h4(dir="auto")= classroom.get('description')
+    h4(dir="auto")!= view.markdownIt(classroom.get('description'))
```

Comment:
What does the `!=` do here? Don't remember seeing that before and I'm curious 😀

Reply:
`!=` means use HTML instead of `=` plain string.